### ISSUE TITLE
Converting question to SQL generates non-working code (extra spaces in db, table, field names)

### ISSUE THREAD

**Issue by sergem155** [ClickHouse/metabase-clickhouse-driver#195]
### Describe the bug
Converting question to SQL generates non-working code (extra spaces in db, table, field names). Sample database (included with Metabase) query does not exhibit this issue
### Steps to reproduce
1. run a new question, e.g. count of rows
2. convert to SQL 
3. click execute

### Expected behaviour
same result as in UI query editor

### Error log

```
generated query:
SELECT
  COUNT(*) AS ` count `
FROM
  ` vd `.` Talk Party `
```

error message:

```
Code: 81. DB::Exception: Database ` vd ` doesn't exist. (UNKNOWN_DATABASE) (version 23.4.2.11 (official build)) , server ClickHouseNode [uri=http://localhost:8123/vd, options={use_server_time_zone_for_dates=true,use_no_proxy=false,product_name=metabase/1.2.1}]@-156336438]
```

#### Environment
* metabase-clickhouse-driver version: 1.2.1
* metabase-clickhouse-driver configuration:
* Metabase version: Metabase v0.47.2 (536c24c release-x.47.x)
* OS: docker (eclipse-temurin) on ubuntu 

#### ClickHouse server
* ClickHouse Server version: 23.4
* ClickHouse Server non-default settings, if any:
* `CREATE TABLE` statements for tables involved:
* Sample data for all these tables, use [clickhouse-obfuscator](https://github.com/ClickHouse/ClickHouse/blob/master/programs/obfuscator/Obfuscator.cpp#L42-L80) if necessary

**Comment by slvrtrn** [comment]
Currently working on it. Thanks for the report.

**Comment by slvrtrn** [comment]
@sergem155, unfortunately, this is a bug in Metabase itself: https://github.com/metabase/metabase/issues/34235

**Comment by Gerrit-K** [comment]
@slvrtrn the bug you linked is closed with the milestone `0.47.7`. I've just upgraded to `0.47.8`, but I'm still seeing these spaces. Is there perhaps an additional issue with the clickhouse driver or should I request to reopen the upstream issue?

**Comment by slvrtrn** [comment]
@Gerrit-K, I thought it would be released as part of 0.48.x (for which I created a branch already: https://github.com/ClickHouse/metabase-clickhouse-driver/pull/204).
I will make a release for 0.47.x that includes part of these changes, then :+1:

**Comment by Gerrit-K** [comment]
@slvrtrn Ah, I see. That would be awesome, thank you!

**Comment by slvrtrn** [comment]
@Gerrit-K @sergem155, fixed in [1.2.4](https://github.com/ClickHouse/metabase-clickhouse-driver/releases/tag/1.2.4).
